src/app/cli/src/init/coda_run.ml

```diff
@@ -494,6 +494,17 @@ let handle_shutdown ~monitor ~conf_dir ~top_logger coda_ref =
 
 You might be trying to connect to a different network version, or need to troubleshoot your configuration. See https://codaprotocol.com/docs/troubleshooting/ for details.
 
+%!|err}
+      | Genesis_ledger_helper.Genesis_state_initialization_error ->
+          Core.eprintf
+            !{err|
+
+  ☠  Coda Daemon failed to initialize the genesis state. The Coda Protocol developers would like to know why!
```

Comment:
I think that this error message pattern could be abstracted some. Nearly every error message follows the same standard whitespace pattern, along with a unicode skull and the same "Coda Daemon failed to" string. The message could be abstracted such that you just specify only the reason and followup instructions (with an optional "The Coda Protocol developers would like to know why!" message).